core/server/data/xml/sitemap/page-generator.js

```diff
@@ -26,6 +26,7 @@ _.extend(PageMapGenerator.prototype, {
             context: {
                 internal: true
             },
+            filter: 'visibility:public',
```

Comment:
question:
There is a `homePage` object manually added in line 34, which never fulfils the condition in `validateDatum`.
Can you explain that to me?